From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Mon, 5 Oct 2015 14:04:39 +0200
Subject: cmap_track_add.3.in: fix typo: bellow -> below

---
 man/cmap_track_add.3.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/cmap_track_add.3.in b/man/cmap_track_add.3.in
index 07a6581..6b6e83e 100644
--- a/man/cmap_track_add.3.in
+++ b/man/cmap_track_add.3.in
@@ -73,7 +73,7 @@ in callback)
 .PP
 .I notify_fn
 is pointer to function which is called when value is changed. It's definition and meaning of parameters
-is discussed bellow.
+is discussed below.
 .I user_data
 argument is passed directly to
 .I notify_fn
