From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Wed, 18 May 2016 23:03:32 +0200
Subject: Fix typo: aquire -> acquire

---
 exec/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/exec/main.c b/exec/main.c
index 457ed09..fa19cd0 100644
--- a/exec/main.c
+++ b/exec/main.c
@@ -1094,7 +1094,7 @@ retry_fcntl:
 			goto error_close;
 			break;
 		default:
-			log_printf (LOGSYS_LEVEL_ERROR, "Corosync Executive couldn't aquire lock. Error was %s",
+			log_printf (LOGSYS_LEVEL_ERROR, "Corosync Executive couldn't acquire lock. Error was %s",
 			    strerror(errno));
 			err = COROSYNC_DONE_AQUIRE_LOCK;
 			goto error_close;
