From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Wed, 18 May 2016 23:06:50 +0200
Subject: Fix typo: retrive -> retrieve

---
 tools/corosync-quorumtool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/corosync-quorumtool.c b/tools/corosync-quorumtool.c
index 3651765..007fd5e 100644
--- a/tools/corosync-quorumtool.c
+++ b/tools/corosync-quorumtool.c
@@ -742,7 +742,7 @@ static int init_all(void) {
 	}
 
 	if (cmap_get_uint32(cmap_handle, "runtime.votequorum.this_node_id", &our_nodeid) != CS_OK) {
-		fprintf(stderr, "Unable to retrive this node nodeid\n");
+		fprintf(stderr, "Unable to retrieve this_node_id\n");
 		goto out;
 	}
 
