From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Wed, 18 May 2016 23:07:08 +0200
Subject: Fix typo: alocated -> allocated

---
 man/cmap_track_add.3.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/cmap_track_add.3.in b/man/cmap_track_add.3.in
index 6b6e83e..0cf9fa8 100644
--- a/man/cmap_track_add.3.in
+++ b/man/cmap_track_add.3.in
@@ -149,7 +149,7 @@ function),
 .I len
 is length of value in cmap and
 .I data
-is pointer to value of item. Data storage is dynamically alocated by caller and notify function must not try to
+is pointer to value of item. Data storage is dynamically allocated by caller and notify function must not try to
 free it.
 
 .SH RETURN VALUE
