From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Wed, 18 May 2016 23:02:41 +0200
Subject: Fix typo: Uknown -> Unknown

---
 exec/sync.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/exec/sync.c b/exec/sync.c
index 7eb4c74..ea452e6 100644
--- a/exec/sync.c
+++ b/exec/sync.c
@@ -328,7 +328,7 @@ static void sync_service_build_handler (unsigned int nodeid, const void *msg)
 			my_service_list[my_service_list_entries].service_id =
 				req_exec_service_build_message->service_list[i];
 			sprintf (my_service_list[my_service_list_entries].name,
-				"Uknown External Service (id = %d)\n",
+				"Unknown External Service (id = %d)\n",
 				req_exec_service_build_message->service_list[i]);
 			my_service_list[my_service_list_entries].sync_init =
 				dummy_sync_init;
