From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Wed, 18 May 2016 23:16:51 +0200
Subject: Fix typo: Diabled -> disabled

---
 exec/totemsrp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/exec/totemsrp.c b/exec/totemsrp.c
index efb7211..1efa729 100644
--- a/exec/totemsrp.c
+++ b/exec/totemsrp.c
@@ -984,7 +984,7 @@ int totemsrp_initialize (
 			log_printf (instance->totemsrp_log_level_debug,
 				"heartbeat_timeout = heartbeat_failures_allowed * token_retransmit_timeout + max_network_delay");
 			log_printf (instance->totemsrp_log_level_debug,
-				"heartbeat timeout should be less than the token timeout. HeartBeat is Diabled !!");
+				"heartbeat timeout should be less than the token timeout. Heartbeat is disabled!!");
 			instance->use_heartbeat = 0;
 		}
 		else {
